backend/src/services/availabilityService.ts

```
@@ -0,0 +1,424 @@
+    }
+
+    return {
+      hasConflict: conflicts.length > 0,
+      conflicts
+    };
+  }
+
+  private static timeRangesOverlap(start1: string, end1: string, start2: string, end2: string): boolean {
+    return start1 < end2 && start2 < end1;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)